pkgvars

```diff
@@ -1,6 +1,6 @@
 pkgname=citus-enterprise
 pkgdesc='Citus Enterprise'
-pkglatest=9.5.6.citus-1
+pkglatest=10.0.4.citus-1
 releasepg=11,12,13
 nightlypg=11,12,13
```

Comment:
```suggestion
nightlypg=12,13
```

`enterprise-master` no longer supports pg11. However I do not think that we create nightly packages for enterprise so this may not have any effect at all.